tests/test_resource_path.py

```diff
@@ -1,28 +1,26 @@
 import sublime
-import shutil
 import tempfile
 
 from sublime_lib import ResourcePath
 from sublime_lib._compat.pathlib import Path
+from sublime_lib._util.temporary_package import TemporaryPackage
```

Comment:
If it's only going to be used for tests, we should move it to the tests package.

Reply:
I'm probably missing something obvious, but I couldn't get UnitTesting to import it because of the weird way that `unittest` loads tests. Is there a simple way around this? I'd definitely move it out to the tests directory if possible.

Reply:
We don't have a `__init__.py` in the tests folder. We should be able to use relative imports with that.

Reply:
Done.